src/mmg3d/mmg3d1.c

```diff
@@ -625,16 +625,24 @@ MMG5_int MMG5_swpmsh(MMG5_pMesh mesh,MMG5_pSol met,MMG3D_pPROctree PROctree, int
 
           ret = MMG5_coquilface(mesh,k,i,ia,list,&it1,&it2,0);
           ilist = ret / 2;
-          if ( ret < 0 )  return -1;
+          if ( ret < 0 ){
+            MMG5_show_tet_location(mesh, pt, k);
+            return -1;
+          }
           /* CAUTION: trigger collapse with 2 elements */
           if ( ilist <= 1 )  continue;
           ier = MMG5_chkswpbdy(mesh,met,list,ilist,it1,it2,typchk);
-          if ( ier <  0 )
+          if ( ier <  0 ){
+            MMG5_show_tet_location(mesh, pt, k);
```

Comment:
Please, keep this in debug mode only (`-d` command line option, `mesh->info.ddebug` variable inside the code) : 
```
if ( mesh->info.ddebug ) {
   MMG5_show_tet_location(mesh, pt, k);
}
```